deploy/manifests/nginx-gateway.yaml

```diff
@@ -124,7 +124,7 @@ spec:
         - static-mode
         - --gateway-ctlr-name=k8s-gateway.nginx.org/nginx-gateway-controller
         - --gatewayclass=nginx
-      - image: nginx:1.23
+      - image: nginx:1.25
         imagePullPolicy: IfNotPresent
```

Comment:
I just realized that `IfNotPresent` makes "This means that the latest available patch version of nginx is used" not true...the version that is already present on the machine will always be used. Should we change this to `Always`?

Reply:
I think so yes